xml/System.Data.SqlClient/SqlAuthenticationMethod.xml

```diff
@@ -11,7 +11,7 @@
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
   <Docs>
-    <summary>Describes the different SQL authentication methods that can be used by a client connecting to Azure SQL Database. For details, see [Connecting to SQL Database By Using Azure Active Directory Authentication](https://azure.microsoft.com/documentation/articles/sql-database-aad-authentication/).</summary>
+    <summary>Describes the different SQL authentication methods that can be used by a client connecting to Azure SQL Database. For details, see [Connecting to SQL Database By Using Azure Active Directory Authentication](/azure/sql-database/sql-database-aad-authentication).</summary>
```

Comment:
The topic title is ...

*Use Azure Active Directory Authentication for authentication with SQL Database or SQL Data Warehouse*